prep-css-box-model/style.css

```diff
@@ -0,0 +1,15 @@
+body {
+  text-align: center;
+}
+
+button {
+  background-color: red;
+  color: white;
+
+  /* 👇 YOUR CODE HERE 👇 */
+  margin: 20px;
+  padding: 10px;
+  border-style: double;
+  border width: 3px;
+  border color: darkgray;color
```

Comment:
This should be `border-color` and the color should only be `darkgray;`, you don't need the word `color` after. The full rule should look like:

```CSS
border-color: darkgray;
```